IEG/IEG-IOC-01App/Db/ieg.db

```diff
@@ -0,0 +1,314 @@
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+    field(PINI, "YES")
+}
+    
+record(bo, "$(P)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+record(bo, "$(P)ABORT")
+{
+    field(DESC, "Send kill signal to device")
+    # Streamdevice doesn't seem to like doing an 'in' command without reading any data, so tell 
+    # it to read device ID and send it to $(P)STATUS:_RAW.A as usual.
+    field(OUT, "@devieg.proto abort($(P)STATUS:_RAW) $(PORT)")
+    field(DTYP, "stream")
+}
+
+record(mbbo, "$(P)MODE:SP")
+{
+    field(DESC, "Operating mode setpoint")
+    field(OUT, "@devieg.proto setMode $(PORT)")
+    field(DTYP, "stream")
+    
+    field(ONVL, "1")
+    field(ONST, "Pump, Purge & Fill")
+    field(TWVL, "2")
+    field(TWST, "Pump")
+    field(THVL, "3")
+    field(THST, "Gas Flow")
+    field(FRVL, "4")
+    field(FRST, "Gas - Single Shot")
+    
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:MODE:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ai, $(P)STATUS:_SCAN)
+{
+    field(INP, "@devieg.proto getStatus($(P)STATUS:_RAW) $(PORT)")
+    field(SCAN, "1 second")
+    field(FLNK, "$(P)STATUS:_RAW")
+    field(DTYP, "stream")
+    
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SCAN")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(calcout, "$(P)STATUS:_RAW")
+{
+    field(CALC, "0")
+    field(FLNK, "$(P)STATUS:_FANOUT")
+}
+
+record(seq, "$(P)STATUS:_FANOUT")
+{
+    field(DOL1, "$(P)STATUS:_RAW.A")
+    field(LNK1, "$(P)ID PP")
+
+    field(DOL2, "$(P)STATUS:_RAW.B")
+    field(LNK2, "$(P)MODE PP")
+
+    field(DOL3, "$(P)STATUS:_RAW.C")
+    field(LNK3, "$(P)VALVESTATE PP")
+
+    field(DOL4, "$(P)STATUS:_RAW.D")
+    field(LNK4, "$(P)ERROR PP")
+
+    field(DOL5, "$(P)STATUS:_RAW.E")
+    field(LNK5, "$(P)PRESSURE:BUFFER:HIGH PP")
+    
+    field(DOL6, "$(P)STATUS:_RAW.F")
+    field(LNK6, "$(P)PRESSURE:LOW PP")
+    
+    field(DOL7, "$(P)STATUS:_RAW.G")
+    field(LNK7, "$(P)PRESSURE:HIGH PP")
+    
+    field(DOL8, "$(P)STATUS:_RAW.H")
+    field(LNK8, "$(P)PRESSURE:RAW PP")
+    
+    field(SELM, "All")
+}
+
+record(ai, "$(P)ID")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(DESC, "ID of this IEG")
+}
+
+record(mbbi, "$(P)MODE")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(DESC, "Operating mode")
+    
+    field(ZRVL, "0")
+    field(ZRST, "Dormant")
+    field(ONVL, "1")
+    field(ONST, "Pump, Purge & Fill")
+    field(TWVL, "2")
+    field(TWST, "Pump")
+    field(THVL, "3")
+    field(THST, "Gas Flow")
+    field(FRVL, "4")
+    field(FRST, "Gas - Single Shot")
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:MODE")
+    field(SDIS, "$(P)DISABLE")    
+    
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(mbbiDirect, "$(P)VALVESTATE")
```

Comment:
https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming

Please could this be `VALVE:STAT`